### DISCUSSION TITLE
有没有英文的简历模板？

### DISCUSSION THREAD

**Discussion by xskxzr** [tuna/thuthesis#626] [Q&A] [answered]
英文论文的简历部分好像要求用英文写，模板里给的是中文例子。另外，中文专利也要用英文写吗？

还一个问题是如果正文用的作者-年份样式的引用，简历部分的论文专利列表依然是数字形式，这合乎要求吗？

**Comment by zepinglee** [comment, marked as answer]
> 英文论文的简历部分好像要求用英文写，模板里给的是中文例子。另外，中文专利也要用英文写吗？

Word 模板中也没给示例。我个人建议用英文写，然后在后面加括号备注下中文。

> 还一个问题是如果正文用的作者-年份样式的引用，简历部分的论文专利列表依然是数字形式，这合乎要求吗？

参考文献跟简历是分开的两套，不冲突。

**Reply by xskxzr** [reply]
主要是专利有国家发布的标准信息，自己翻译总感觉有点不标准，就像英文文章也可以引用中文文献，直接专利写中文应该没问题吧？

**Reply by Harry-Chen** [reply]
> 主要是专利有国家发布的标准信息，自己翻译总感觉有点不标准，就像英文文章也可以引用中文文献，直接专利写中文应该没问题吧？

请咨询格式审查相关老师，这个问题我们无法提供回答
